app/views/bids/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="row col-md-8 col-md-offset-2">
+  <h1><%= @bid.project.service.name %></h1>
+
+  <div class="panel panel-default">
+    <% if @bid.messages.any? %>
```

Comment:
What happens if we remove this if statement? I think it should be OK since rendering a partial with an empty array should just render nothing.